indico/modules/events/abstracts/templates/abstract/timeline.html

```diff
@@ -144,10 +152,12 @@
                                 {%- endtrans %}
                             {% endif %}
                         {% endif %}
-                    </div>
-                    <div class="abstract-comment-score">
                         {% if review.score is not none %}
-                            {{ render_score(review.score, review.abstract.event_new.cfa) }}
+                            · score {{ render_score(review.score, review.abstract.event_new.cfa, classes='bold semantic-text') }}
+                            (<a class="js-ratings-toggle">{#--#}
```

Comment:
What is the `{#--#}` doing exactly? Don't you want to put the `-#}` in the next line to remove the whitespace?